### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AliaksandrZahorski**
Базовые функции калькулятора:
сложение, вычитание, умножение, деление, целочисленное деление

### CODE REVIEW

**Comment by NikitaSedyx** [comment]
hi, @AliaksandrZahorski could you make a separate folder for this task? thank you
basically code is fine, please try how it works without arguments, with `null` argument, strings like `ababa`

**Comment by AliaksandrZahorski** [comment]
Sure. Now code in separate folder and new tests are added.

File: functions/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
```
Comment (NikitaSedyx) [inline functions/index.html]: please use 2 spaces everywhere instead of 4

File: functions/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+</head>
+<body>
+	Hello!
+	<script src="script.js"></script>
+</body>
+</html>
```
Comment (NikitaSedyx) [inline functions/index.html]: empty line please at the EOF

File: functions/script.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+console.log(plus(1, true));
+console.log(div(10, 3));
+console.log(plus());//undefined->NaN
+console.log(plus(1));//undefined->NaN
+console.log(plus(null, null));//null->0
+console.log(plus("test ", "me"));
+console.log(plus("test ", "me", "again"));
+
+function plus(a, b) {
+	var result = a + b;
```
Comment (NikitaSedyx) [inline functions/script.js]: the same for js

File: functions/script.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	return result;
+}
+
+function minus(a, b) {
+	var result = a - b;
```
Comment (NikitaSedyx) [inline functions/script.js]: actually you could do `return a - b;`.  less code + same sense
the same for another functions.

**Review by NikitaSedyx — CHANGES_REQUESTED** [review]

File: functions/index.html [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+</head>
+<body>
```
Comment (NikitaSedyx) [inline functions/index.html]: please make indentations for each nested tags

File: functions/index.html (hunk 1 above)
Comment (AliaksandrZahorski) [reply functions/index.html]: done

File: functions/index.html (hunk 2 above)
Comment (AliaksandrZahorski) [reply functions/index.html]: add line

File: functions/script.js (hunk 3 above)
Comment (AliaksandrZahorski) [reply functions/script.js]: fulfilled

File: functions/index.html (hunk 5 above)
Comment (AliaksandrZahorski) [reply functions/index.html]: performed

**Comment by AliaksandrZahorski** [comment]
Thanks for the helpful comments!

**Review by NikitaSedyx — APPROVED** [review]

File: functions/script.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+function multiply(a, b) {
+  return a * b;
+}
+
+function divide(a, b) {
+  return a / b;
+}
+
+function div(a, b) {
+  return (a - a % b) / b;
```
Comment (NikitaSedyx) [inline functions/script.js]: `%` sometimes returns result you don't want
please try `3.3 % 2`.
if I will pass float number in your function in can give me wrong result, please test it

Comment (AliaksandrZahorski) [reply functions/script.js]: More tests are added. Also new version of function "divT(a, b)" is added.
Looks like they both returns expected values.

File: functions/script.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+function divT(a, b) {
+  return trunc(a/b);;
+}
+
+function trunc(x) {
+  if (isNaN(x)) {
```
Comment (NikitaSedyx) [inline functions/script.js]: this `if` doesn't make sense, Math.ceil(NaN) = NaN

File: functions/script.js [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+function divide(a, b) {
+  return a / b;
+}
+
+function div(a, b) {
+  return (a - a % b) / b;
+}
+
+function divT(a, b) {
+  return trunc(a/b);;
```
Comment (NikitaSedyx) [inline functions/script.js]: two semicolons?

Comment (NikitaSedyx) [reply functions/script.js]: you could use parseInt :) it's safer than (/ - / %)

**Review by NikitaSedyx — APPROVED** [review]
please fix comments and merge

**Pull request merged**